#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# (c) ACE 

import os

class Config(object):
    # get a token from @BotFather
    pass
    """
    BOT_TOKEN = os.environ.get("BOT_TOKEN", "7126892855:AAGg__RShjmnJCmr501S3Y24hOVmj8dGdMo")
    API_ID = int(os.environ["API_ID", "28308147"]
    API_HASH = os.environ["API_HASH", "8240cdc945560ccd108b6ff2925ced5e"]
    AUTH_USERS = "5618212564"""
# jisko bhi is repo ko deploy karne me error aa raha ho ( https://t.me/Stubborn1223 ) ye mere TG ka id ha message karke iska screen shot send kar dena me error solve kar dunga. 
